main_test.go

```diff
@@ -90,6 +90,37 @@ func TestGofmt(t *testing.T) {
 	}
 }
 
+func TestGoPyErrors(t *testing.T) {
+	pyvm := testBackends["py3"]
+	workdir, err := ioutil.TempDir("", "gopy-")
+	if err != nil {
+		t.Fatalf("could not create workdir: %v\n", err)
+	}
+	fmt.Printf("pyvm: %s making work dir: %s\n", pyvm, workdir)
+	//defer os.RemoveAll(workdir)
+
+	curPkgPath := reflect.TypeOf(pkg{}).PkgPath()
+
+	// fmt.Printf("building in work dir: %s\n", workdir)
+	fpath := filepath.Join(curPkgPath, "_examples/gopyerrors")
+	args := []string{"run", ".", "gen", "-vm=" + pyvm, "-output=" + workdir, fpath}
+	cmd := exec.Command("go", args...)
+	fmt.Printf("running: %v\n", args)
```

Comment:
```suggestion
	t.Logf("running: %v\n", cmd.Args)
```
(and perhaps remove the `args` var alltogether?)